node/src/types/block.rs

```diff
@@ -582,7 +597,7 @@ pub struct BlockHeader {
     transfer_hashes: Vec<DeployHash>,
     random_bit: bool,
     accumulated_seed: Digest,
-    era_end: Option<EraEnd>,
```

Comment:
This changes the structure of a `BlockHeader`, I thought we didn't want to do it for delta.

Reply:
[`4375f11` (#943)](https://github.com/CasperLabs/casper-node/pull/943/commits/4375f11ef896eaa788492e5ae03179b80c79846f) also changes the structure

Reply:
I believe this _reverts_ a change that was on `master`, but not on the release branch... doesn't it?

Reply:
Although... it looks like it was called `EraEnd` on the original release branch? So maybe the name shouldn't change.

Reply:
This PR is against the old release (Delta) branch so we will see the changes to the delta.

Reply:
Yes, exactly